testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -101,9 +102,9 @@ public void TestRunTeardown()
         m_TestResultsLocal.Clear();
     }
 
-    public void OnDestroy()
+    public void OnDisable()
```

Comment:
OnClientDisconnectCallback is added in Start()
Start is called only once.
For symmetry, you should also switch the subscription to this callback to OnEnable

Reply:
Please confirm that this is now correct

Reply:
yep this is good